libnmstate/varlink/__init__.py

```diff
@@ -0,0 +1,292 @@
+#
```

Comment:
Normally, we don't put real code into `__init__.py`. Please create another python code file and import needed into `__init__.py`.